tests/input_flows.py

```diff
@@ -385,6 +385,32 @@ def input_flow_tr(self) -> GeneratorType:
         self.debug.press_middle()
 
 
+class InputFlowShowXpubQRCode(InputFlowBase):
+    def __init__(self, client: Client):
+        super().__init__(client)
+
+    def input_flow_tt(self) -> GeneratorType:
+        yield
+        self.debug.click(buttons.CORNER_BUTTON, wait=True)
+        # synchronize; TODO get rid of this once we have single-global-layout
+        self.debug.synchronize_at("HorizontalPage")
+
+        self.debug.swipe_left(wait=True)
+        self.debug.swipe_right(wait=True)
+        self.debug.swipe_left(wait=True)
```

Comment:
Do we want to swipe so many times to show both the QR code and account information twice? Theoretically just one `swipe_left` would be enough to show both of them

Reply:
What do you mean by "so many times"?

You should see xpub (swipe left) qr code (swipe left) account info

Reply:
These are the test-case screens, we are going twice there and back through the QR code and details, the middle screens do not show any new information. Maybe it is testing that we can successfully go back and click corner button from the second screen? It is fully valid, just wondering

![image](https://github.com/trezor/trezor-firmware/assets/42543243/073492aa-dece-466a-bd8a-fe5d710b75fb)

Reply:
🤷🏻

Reply:
Yes, it's meant to test the code for going back. May be already covered by the receive flow testcase but it costs us almost nothing.